README.md

```diff
@@ -512,6 +512,10 @@ axios.defaults.headers.common['Authorization'] = AUTH_TOKEN;
 axios.defaults.headers.post['Content-Type'] = 'application/x-www-form-urlencoded';
 ```
 
+Note, if you assign data to the request, `Content-Type` wouldn't adjust to default header setting. This is because `Content-Type` must be `application/json` when transmitting data.
```

Comment:
Yes!  As [more-stories-about-headers](https://github.com/chinesedfan/You-Dont-Know-Axios#more-stories-about-headers) say default Content-Type setting  is reset, if there would be config.data.

I think It is duplicate content.

Reply:
> Content-Type will be removed if data is FormData or undefined, in order to let the browser to set.

Note the differences between your PR and my document. If `config.data` is something else, it will not be removed.

Reply:
I understand now!
I fixed description and refer your document for more detail.